packages/material-ui-codemod/README.md

```diff
@@ -349,6 +350,21 @@ npx @material-ui/codemod@next v5.0.0/collapse-rename-collapsedheight <path>
 
 You can find more details about this breaking change in [the migration guide](https://next.material-ui.com/guides/migration-v4/#collapse).
 
+#### `color-imports`
```

Comment:
I would propose a different approach here. This new codemod is interesting but not leveraging the potential of the root issue we have. In v4, we have introduced a codemod https://github.com/mui-org/material-ui/blob/next/packages/material-ui-codemod/README.md#optimal-imports to migrate ALL these private import instances. Why did we do it? Because it was leading to duplication of modules in the bundles, and breaking the JSS style injection orders.

This codemod is not working with v5 anymore. In v5 we still want to forbid private imports. The difference compared to v4 is that now, most private imports fail and fast, in v4 it was insidious. 

So I would encourage we start from `optimal-imports`, we fix it to work with v5 bundle, and we rename it to `private-import` or something similar. optimal suggests that it's optional, it's not. It will be even more true with #26254.

Reply:
> This codemod is not working with v5 anymore

In which case that `optimal-imports` does not work anymore?

Reply:
When I try to run it, it fails on something related to the `/es` folder that we had in v4.

Reply:
@oliviertassinari can we merge this and improve later? I don't have enough bandwidth to do the change.

Reply:
or if @mnajdova can take over this.

Reply:
Taking a look

Reply:
Sorry didn't got to this today, will take a look tomorrow. Need to go over the comments once again :)

Reply:
Let's resolve https://github.com/mui-org/material-ui/issues/27320 first as I can't really test anything...

Reply:
Created https://github.com/mui-org/material-ui/pull/27404 we have a problem with some private imports which are kind of consider public but they are not. There is more info on the PR description. The colors are going to be handled by this new codemod automatically, so if we go with that option we can close this one.

Reply:
> if we go with that option we can close this one.

We update the docs here too. Maybe we could focus only on this in this PR?

Reply:
Let's do that.

Reply:
I've reverted all codemode related changes from this one, and moved them to https://github.com/mui-org/material-ui/pull/27404